cmd/lncli/commands.go

```diff
@@ -215,14 +221,18 @@ func sendCoins(ctx *cli.Context) error {
 		amt = ctx.Int64("amt")
 	case args.Present():
 		amt, err = strconv.ParseInt(args.First(), 10, 64)
-	default:
+	case !ctx.Bool("sweepall"):
 		return fmt.Errorf("Amount argument missing")
 	}
-
 	if err != nil {
 		return fmt.Errorf("unable to decode amount: %v", err)
 	}
 
+	if amt != 0 && ctx.Bool("sweepall") {
+		return fmt.Errorf("amount cannot be set if attempting to " +
+			"sweep all coins out of the wallet")
+	}
+
```

Comment:
should we add a user confirmation for sweep all? i would hate for someone to accidentally do this and send all their coins to an address they don't control. that could partially be addressed by using a different rpc method altogether, though i think the confirmation is a happy medium